WinApi/Windows/NativeWindow.cs

```diff
@@ -43,7 +43,7 @@ public string GetText()
             if (size > 0)
             {
                 var sb = new StringBuilder(size);
```

Comment:
Ah. You're right. My bad, replied being half sleepy. `nMaxCount` is just the number of characters. For some reason, had it mind that it was the buffer size.